modules/nw-scaling-ingress-controller.adoc

```diff
@@ -52,8 +57,19 @@ $ oc get -n openshift-ingress-operator ingresscontrollers/default -o jsonpath='{
 ----
 3
 ----
-
-[NOTE]
++
+[TIP]
 ====
-Scaling is not an immediate action, as it takes time to create the desired number of replicas.
+You can alternatively apply the following YAML to scale an Ingress Controller:
```

Comment:
Maybe clarify this by pointing out the change is to the `replicas` value, e.g.:
You can alternatively apply the following YAML to scale an Ingress Controller to three replicas.  If you desire fewer or more than three, change the `replicas` value.